ReactiveCocoa/Association.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  Association.swift
+//  Rex
+//
+//  Created by Neil Pankey on 6/19/15.
+//  Copyright (c) 2015 Neil Pankey. All rights reserved.
+//
+
+import ReactiveSwift
+
+/// Attaches a `MutableProperty` value to the `host` object using KVC to get the initial
+/// value and write subsequent updates from the property's producer. Note that `keyPath`
+/// is a `StaticString` because it's pointer value is used as key value when associating
+/// the property.
+///
+/// This can be used as an alternative to `DynamicProperty` for creating strongly typed
+/// bindings on Cocoa objects.
+public func associatedProperty(_ host: AnyObject, keyPath: StaticString) -> MutableProperty<String> {
```

Comment:
Maybe we should make these `internal` for now?

I'd rather be conservative in terms of API and encourage people to contribution missing items.